components/PaginatedSuppliers/Paginated.module.scss

```diff
@@ -0,0 +1,96 @@
+@import 'colors','fonts','layout';
+
+.paginated-container {
+  align-items: center;
+  bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+  margin: 10px 50px;
+  max-height: 43px;
+  padding: 20px auto;
+  position: fixed;
+  width: 1085px;
```

Comment:
Is this the correct width? It doesn't look good on my screen
![image](https://user-images.githubusercontent.com/61360270/188316511-859faa4f-842a-451a-93dc-164daa457a07.png)